src/o365/teams/commands/report/teams-report-deviceusagedistributionusercounts.spec.ts

```diff
@@ -141,6 +318,28 @@ describe(commands.TEAMS_REPORT_DEVICEUSAGEDISTRIBUTIONUSERCOUNTS, () => {
     });
   });
 
+  it('supports specifying outputFilePath', () => {
+    const options = (command.options() as CommandOption[]);
+    let containsOption = false;
+    options.forEach(o => {
+      if (o.option.indexOf('--outputFilePath') > -1) {
+        containsOption = true;
+      }
+    });
+    assert(containsOption);
+  });
+
+  it('supports specifying output', () => {
```

Comment:
I don't think we need this test here since we're not introducing a new output type and json and text are already covered in the base class.

Reply:
Done. This is fixed